mailpoet/assets/js/src/homepage/components/homepage-sections.tsx

```diff
@@ -26,6 +27,9 @@ export function HomepageSections(): JSX.Element {
           <ProductDiscovery onHide={hideProductDiscovery} />
         </ErrorBoundary>
       ) : null}
+      <ErrorBoundary>
+        <SubscribersStats />
+      </ErrorBoundary>
```

Comment:
I have a little understanding of why you are doing this, but I want to confirm. Do we need so many Error Boundaries?

We can also compress this line down to
```suggestion
          {!isProductDiscoveryHidden && (
        <ErrorBoundary>
          <ProductDiscovery onHide={hideProductDiscovery} />
        </ErrorBoundary>
      )}
      <ErrorBoundary>
        <SubscribersStats />
      </ErrorBoundary>
```

Reply:
> I have a little understanding of why you are doing this, but I want to confirm. Do we need so many Error Boundaries?

@triple0t I forgot to answer this question. I thought it might be a good idea to wrap each section into a boundary so that when one section breaks down with an error, the other will still be available for the user. I see those sections as independent widgets so it makes sense to me to wrap them like that. But it is not any super strong opinion.

Reply:
I understand.